### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by MonaxGT**
Hi!
I add docker support for your application and rewrote requirements.txt. All works if you run:

```
docker build . -t twitter-intelligence
docker run -it -p 5000:5000 --rm -v "images:/usr/src/app/images" twitter-intelligence
```

You can drop option -p if it isn't required

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,9 @@
-lxml==4.2.1
-pyquery==1.4.0
-Flask==0.12.3
-pandas==0.22.0
-termcolor==1.1.0
-geopy==1.11.0
-requests==2.19.1
-numpy==1.14.1
-matplotlib==2.2.2
-PyQt5==5.11.2
```
Comment (batuhaniskr) [inline requirements.txt]: Hi,

Why removed to PyQt5 from requirements.txt file?

**Comment by MonaxGT** [comment]
I had some troubles when built docker with PyQt from requirements.txt.  I added with package on Dockerfile 
```

**Comment by MonaxGT** [comment]
I am sorry, i close request by chance

**Comment by batuhaniskr** [comment]
Okey. You can add new docker-requirements.txt named requirements without PyQt. The existing file may remain attached to PyQt module.

**Comment by batuhaniskr** [comment]
I converted PyQt module to comment line in requirements.txt for that problem. Thank you for contribution.

**Pull request merged**